command/connect/envoy/exec_windows.go

```diff
@@ -0,0 +1,121 @@
+//go:build windows
```

Comment:
I see some duplicate code between exec_windows.go and exec_unix.go, wondering if we can add an exec_supported file which will have the shared code between windows and unix?